### PR TITLE
sha3 functions

### PR DESCRIPTION
**Pull request by domel**

### CODE REVIEW

File: SEP/SEP-0008/sep-0008.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+* SEP Number: [SEP-0008](sep-0008.md)
+* Author: [Dominik Tomaszuk](https://github.com/domel)
+
+## Authors
+
+Dominik Tomaszuk
+
+## Abstract
+
+SHA-3 (Secure Hash Algorithm 3) is the member of the Secure Hash Algorithm family of standards, released by NIST on August 5, 2015. This document presents a set of SHA-3 functions in SPARQL.
```
Comment (TallTed) [inline SEP/SEP-0008/sep-0008.md:13]:
```suggestion
SHA-3 (Secure Hash Algorithm 3) is a member of the Secure Hash Algorithm family of standards, released by NIST on August 5, 2015. This document presents a set of SHA-3 functions in SPARQL.
```

File: SEP/SEP-0008/sep-0008.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Motivation
+
+SPARQL was published in 2013. In 2015 a new set of SHA-3 functions appeared. The SHA-3 functions have several benefits:
```
Comment (TallTed) [inline SEP/SEP-0008/sep-0008.md:17]:
```suggestion
SPARQL was published in 2013. In 2015, a new set of SHA-3 functions appeared. The SHA-3 functions have several benefits:
```

File: SEP/SEP-0008/sep-0008.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+* Security: SHA-3 functions provide a high level of security against various types of attacks such as pre-image, collision, and length extension attacks. 
+* Popularity: SHA-3 functions are widely used.
+* Efficiency: SHA-3 functions are optimized for performance and can be implemented efficiently in software. They are designed to be faster and more efficient than their predecessors, such as SHA-1 and SHA-2.
+* Standardization: SHA-3 functions are standardized by NIST and have been extensively reviewed by the cryptographic community. 
+* Future-proofing: SHA-3 functions were designed with the future in mind, and they have a long projected lifespan. They are resistant to quantum computing attacks, making them an excellent choice for long-term security needs.
```
Comment (TallTed) [inline SEP/SEP-0008/sep-0008.md:23]:
```suggestion
* **Security:** SHA-3 functions provide a high level of security against various types of attacks such as pre-image, collision, and length extension attacks. 
* **Popularity:** SHA-3 functions are widely used.
* **Efficiency:** SHA-3 functions are optimized for performance and can be implemented efficiently in software. They are designed to be faster and more efficient than their predecessors, such as SHA-1 and SHA-2.
* **Standardization:** SHA-3 functions are standardized by NIST and have been extensively reviewed by the cryptographic community. 
* **Future-proofing:** SHA-3 functions were designed with the future in mind, and they have a long projected lifespan. They are resistant to quantum computing attacks, making them an excellent choice for long-term security needs.
```

File: SEP/SEP-0008/sep-0008.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+* `sha3_224` – 28 bit Digest-Size
+* `sha3_256` – 32 bit Digest-Size
+* `sha3_384` – 48 bit Digest-Size
+* `sha3_512` – 64 bit Digest-Size
+
+
+
+
+
```
Comment (TallTed) [inline SEP/SEP-0008/sep-0008.md:35]:
```suggestion
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by TallTed — APPROVED** [review]

**Comment by JervenBolleman** [comment]
Looks good to me. Could the section be extended with examples as in [the hash section](https://www.w3.org/TR/sparql11-query/#func-hash) of the current spec?

**Review by afs — APPROVED** [review]

**Pull request merged**